modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/builder/SchemaBuilder.java

```diff
@@ -706,4 +707,17 @@ void addInlineParameterGroup(ParameterGroupModel group, ExplicitGroup parentSequ
     complexType.getComplexContent().getExtension().setSequence(groupSequence);
     parentSequence.getParticle().add(objectFactory.createElement(groupElement));
   }
+
+  String getNamespaceUri(String prefix) {
+    if (prefix.equals(CORE_PREFIX))
+      return CORE_NAMESPACE;
+    if (prefix.equals(EE_PREFIX))
+      return EE_NAMESPACE;
+    Optional<ExtensionModel> extensionModelFromPrefix = dslContext.getExtensions().stream()
+        .filter((extensionModel) -> prefix.equals(extensionModel.getXmlDslModel().getPrefix())).findFirst();
+    if (extensionModelFromPrefix.isPresent()) {
+      return extensionModelFromPrefix.get().getXmlDslModel().getNamespace();
```

Comment:
this is not right, we never check that the namespace returned was actually imported by the schema. The "ee" namespace is not being imported in the schema either (check the xsd). 
- import the namespace if it was not imported before 
- add a test that validates this imports 
- verify at least manually that the substitutiongroup.xsd is a valid schema. Better yet to actually use the extension in some xml that gets parsed and executed by mule